cmd/main.go

```diff
@@ -7,23 +7,29 @@ import (
 	"flag"
 	"os"
 	"strings"
+	"time"
+
+	"context"
```

Comment:
Nit: Why the linebreak between this and the previous ones?

Reply:
fixed